hail/python/benchmark/hail/benchmark_combiner.py

```diff
@@ -0,0 +1,77 @@
+from test.hail.helpers import with_flags
+
+import pytest
+
+import hail as hl
+from benchmark.tools import benchmark, chunk
+from hail.vds.combiner import combine_variant_datasets, new_combiner, transform_gvcf
+
+COMBINE_GVCF_MAX = 100
+MAX_TO_COMBINE = 20 * COMBINE_GVCF_MAX
+
+
+def import_vcf(path):
+    return hl.import_vcf(str(path), reference_genome='GRCh38', force=True)
+
+
+@benchmark()
+@with_flags(no_ir_logging='1')
+def benchmark_compile_2k_merge(empty_gvcf, tmp_path):
+    vcf = import_vcf(empty_gvcf)
+    vcfs = [transform_gvcf(vcf, [])] * COMBINE_GVCF_MAX
+    combined = [combine_variant_datasets(vcfs)] * 20
+    hl.vds.write_variant_datasets(combined, str(tmp_path / 'combiner-multi-write'), overwrite=True)
+
+
+@benchmark()
+def benchmark_python_only_10k_transform(empty_gvcf):
+    vcf = import_vcf(empty_gvcf)
+    vcfs = [vcf] * 10_000
+    _ = [transform_gvcf(vcf, []) for vcf in vcfs]
+
+
+@benchmark()
+def benchmark_python_only_10k_combine(empty_gvcf):
+    vcf = import_vcf(empty_gvcf)
+    mt = transform_gvcf(vcf, [])
+    mts = [mt] * 10_000
+    _ = [combine_variant_datasets(mts) for mts in chunk(COMBINE_GVCF_MAX, mts)]
```

Comment:
Sure I'll change this.
FWIW, the body of most of these benchmarks has been left more or less intact but sadly there have been too many changes to the rest of the file that git thinks that these are a delete/insert rather than rename/modify.

Reply:
I tried to separate out these changes in another PR but the squash/rebase makes that work redundant :(